src/appleseed/renderer/kernel/rendering/final/uniformpixelrenderer.cpp

```diff
@@ -181,7 +183,7 @@ namespace
                     m_total_sampling_dim.insert(child_sampling_context.get_total_dimension());
 
                     // Merge the sample into the framebuffer.
-                    if (shading_result.is_valid())
+                    if (shading_result.is_valid() && shading_result.m_main.r < 0.8f)
```

Comment:
Shouldn't be part of the PR :)

Reply:
Woops, that's weird, I used `--patch` `to add and was careful.